backend/app/models/assistants.py

```
@@ -27,7 +30,63 @@ class Assistant(AssistantBase, table=True):
     # Relationships
     project: "Project" = Relationship(back_populates="assistants")
     organization: "Organization" = Relationship(back_populates="assistants")
+
+
+class AssistantCreate(BaseModel):
+    name: str = PydanticField(description="Name of the assistant")
+    instructions: str = PydanticField(description="Instructions for the assistant")
+    model: str = PydanticField(
+        default="gpt-4o", description="Model name for the assistant"
+    )
+    vector_store_ids: List[str] = PydanticField(
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)